### DISCUSSION TITLE
How to enable/disable hand and gaze in MRTK3

### DISCUSSION THREAD

**Discussion by amea32179** [MixedRealityToolkit/MixedRealityToolkit-Unity#462] [General]
@AMollis , @marek-stoj @marlenaklein-msft
Hi, Is there a way to mimic the implementation of SetGazePointerBehavior, SetHandPokePointerBehavior, SetHandGrabPointerBehavior and SetPointerBehavior in MRTK 3?
![image](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/138874872/611f7d47-b387-4bff-82ef-7a1396c39cd6)

**Comment by AMollis** [comment]
@amea32179 , you just need to disable the XRI interactor you want disabled.   For example, disable the the "GazeInteractor" `GameObject` to disable gaze interaction:

![image](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/36461279/4f97e4b7-d01e-4db0-a002-4272bf347fa0)

For an example of how to disable interactors at runtime,, see the [DirectionalIndicatorExample.unity](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/blob/main/UnityProjects/MRTKDevTemplate/Assets/Scenes/DirectionalIndicatorExample.unity) sample scene. This scene uses the [`InteractorBehaviorControls`](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/blob/main/UnityProjects/MRTKDevTemplate/Assets/Scripts/InteractorBehaviorControls.cs) sample component to disable and enable interactor `GameObjects`.
